key.core/src/main/antlr4/KeYParser.g4

```diff
@@ -442,8 +450,16 @@ term
  */
 accessterm
 :
-  (sortId DOUBLECOLON)?
-  firstName=simple_ident
+  // OLD
```

Comment:
Rewrite of `acessterm`. It is more uglier, but requires no lookahead. 

I am not sure whether we should do it.